lib/api/bulk.js

```diff
@@ -32,13 +32,16 @@ var inherits     = require('inherits'),
  * @param {Object} [options] - Options for bulk loading operation
  * @param {String} [options.extIdField] - External ID field name (used when upsert operation).
  * @param {String} [options.concurrencyMode] - 'Serial' or 'Parallel'. Defaults to Parallel.
+ * @param {String} [options.contentType] - The content type of the data.
+ *  Possible values 'CSV', 'XML', 'JSON', 'ZIP_CSV', 'ZIP_XML', or 'ZIP_JSON'. Defaults to 'CSV'.
  * @param {String} [jobId] - Job ID (if already available)
```

Comment:
All the possible values the contentType can be according to Salesforce documentation

Reply:
You might consider throwing these values into an enum somewhere. That would also afford you to the opportunity to verify that the given option is a valid one, and throw an error if it's not.